react.js

```diff
@@ -5,8 +5,16 @@
 
 // 1. Write a React component that prints "I am a component!" Be sure to include all necessary imports, exports, etc.
 
+import React, { Component } from 'react'
 
```

Comment:
class Print extends component
"I am a component!" should be wrapped in a JSX tag